config.properties.template

```diff
@@ -121,6 +125,9 @@ display_pokestop_name=false
 # Should the bot auto transfer duplicate pokemon
 autotransfer=true
 
+# Should the bot wait between each transfer and how long? -1 to disable, time to wait in milliseconds (it will be randomized alittle around the given value)
```

Comment:
What would be a realistic (= ingame) value for this? Maybe put that value in this description so people can choose to use this value?